### ISSUE TITLE
TL-UL Verification

### ISSUE THREAD

**Issue by shahzaibk23** [merledu/caravan#13]
Need to verify the Tilelink Uncached Lightweight implementation in Caravan.
- Refer to the Tilelink [Specs](https://static.dev.sifive.com/docs/tilelink/tilelink-spec-1.7-draft.pdf) for TL-UL Protocol.
- Match the Specs to the TL-UL Implementation in [Caravan](https://github.com/merledu/caravan/tree/main/src/main/scala/caravan/bus/tilelink).

**Comment by SyedHassanUlHaq** [comment]
I am in

**Comment by TiwariAbhishek23** [comment]
I am in

**Comment by shahzaibk23** [comment]
Let me know if you guys have any questions regarding this issue. @AbhishekTiwari23 @SyedHassanUlHaq

**Comment by TiwariAbhishek23** [comment]
Sir, till when we have to complete it

**Comment by TiwariAbhishek23** [comment]
Because of the code forces contest, today I was a little bit busy so that I will start it tomorrow

**Comment by TiwariAbhishek23** [comment]
And, we have to read the pin details in the code, match it with the provided documentation, and understand it. Right?

**Comment by shahzaibk23** [comment]
Yes, match it and provide a report if anything in implementation looks not a/c to Specs.
